src/frontend/screens/Settings/sections/GamesSettings/index.tsx

```diff
@@ -139,6 +140,11 @@ export default function GamesSettings() {
       }
       getIsNative()
     }
+    const getIsVulkanSupported = async () => {
+      const isSupported = await window.api.isVulkanSupported()
+      setIsVulkanSupported(isSupported)
+    }
+    getIsVulkanSupported()
```

Comment:
since the vulkan support cannot change at runtime, we could expose that information at boot time in the preload script like we do here https://github.com/Heroic-Games-Launcher/HeroicGamesLauncher/blob/main/src/backend/preload.ts#L7

it makes the handling of this information way easier without having to deal with ipc messages and async calls, just read from windows